fhir-parent/pom.xml

```diff
@@ -225,7 +225,7 @@
                  <!-- There are breaking changes in 10.15+ don't upgrade beyond this -->
                 <groupId>org.apache.derby</groupId>
                 <artifactId>derbyclient</artifactId>
-                <version>10.14.2.0</version>
+                <version>10.15.2.0</version>
```

Comment:
Since this isn't derby (rather just the client), I think this is OK, however do we want to keep these lined up?